### PR TITLE
chore: integrate next.js package properly

### PR DESCRIPTION
**Pull request by jonas-jonas**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: examples/nextjs-app-router/app/auth/login/page.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Login } from "@ory/elements-react"
+import { Login } from "@ory/elements-react/theme"
```
Comment (aeneasr) [inline examples/nextjs-app-router/app/auth/login/page.tsx:1]: Should this be `theme/default`?

File: packages/nextjs/project.json [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "packages/nextjs/src",
   "projectType": "library",
   "tags": [],
   "targets": {
     "build": {
-      "executor": "@nx/js:tsc",
-      "outputs": ["{options.outputPath}"],
+      "command": "tsup --clean --dts",
+      // "dependsOn": ["lint", "test"],
```
Comment (aeneasr) [inline packages/nextjs/project.json:10]: Invalid JSON?

**Review by aeneasr — APPROVED** [review]

File: examples/nextjs-app-router/app/auth/login/page.tsx (hunk 1 above)
Comment (aeneasr) [reply examples/nextjs-app-router/app/auth/login/page.tsx:1]: Ok yeah, that makes sense. So then the CSS import change also needs to be reverted

Comment (jonas-jonas) [reply examples/nextjs-app-router/app/auth/login/page.tsx:1]: I never merged it. https://github.com/ory/elements/pull/241

**Pull request merged**